doc/rados/configuration/ceph-conf.rst

```diff
@@ -620,84 +623,84 @@ Override values can be set in two ways:
       ceph daemon osd.4 config set debug_osd 20
 
 .. note:: In the output of the ``ceph config show`` command, these temporary
-   values are shown with a source of ``override``.
+   values are shown to have a source of ``override``.
 
 
 Viewing runtime settings
 ========================
 
-You can see the current options set for a running daemon with the ``ceph config show`` command.  For example:
+You can see the current settings specified for a running daemon with the ``ceph
+config show`` command. For example, to see the (non-default) settings for the
+daemon ``osd.0``, run the following command:
 
 .. prompt:: bash $
 
    ceph config show osd.0
 
-will show you the (non-default) options for that daemon.  You can also look at a specific option with:
+To see a specific setting, run the following command:
 
 .. prompt:: bash $
 
    ceph config show osd.0 debug_osd
 
-or view all options (even those with default values) with:
+To see all settings (even those with default values), run the following
```

Comment:
s/even/including/

Reply:
Another good suggestion of the word "including". Accepted.